apis/r/vignettes/soma-experiment-queries.Rmd

```diff
@@ -41,6 +41,13 @@ We can see that our current experiment contains only a single measurement: `"RNA
 experiment$ms
 ```
 
+To use larger (or smaller) buffer sizes:
```

Comment:
Maybe mention the environment variable setter as the usual fallback (as we are going to Core code here) ?